frontend/packages/app/package.json

```diff
@@ -12,7 +12,7 @@
     "lint": "yarn register-workflows && eslint --ext .js,.jsx .",
     "lint:fix": "yarn run lint --fix",
     "register-workflows": "yarn workspace @clutch-sh/tools registerWorkflows $PWD/src",
-    "start": "yarn register-workflows && react-app-rewired start",
+    "start": "yarn register-workflows && GENERATE_SOURCEMAP=false react-app-rewired start",
```

Comment:
Disabled sourcemaps for CRA as it now throws an error if it cant find them for a few packages, very annoying error.

Updating react-router and react-router-dom fixes 2 of them but the other would require a bit more effort.

```
WARNING in ../../node_modules/react-double-scrollbar/dist/DoubleScrollbar.js
Module Warning (from ../../node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '<clutch>/frontend/node_modules/react-double-scrollbar/dist/DoubleScrollbar.js.map' file: Error: ENOENT: no such file or directory, open '<clutch>/frontend/node_modules/react-double-scrollbar/dist/DoubleScrollbar.js.map'
 @ ../../node_modules/material-table/dist/material-table.js 42:51-84
 @ ../../node_modules/material-table/dist/index.js 29:44-71
 @ ../core/dist/Table/tree-table.js 11:0-43 68:45-58
 @ ../core/dist/Table/index.js 6:0-37 7:0-16:2
 @ ../core/dist/index.js 30:0-39:17 49:0-102:2 49:0-102:2 49:0-102:2 49:0-102:2 49:0-102:2 49:0-102:2 49:0-102:2 49:0-102:2
 @ ./src/index.jsx 5:0-42 9:52-61

WARNING in ../../node_modules/react-router-dom/index.js
Module Warning (from ../../node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '<clutch>/frontend/packages/react-router-dom/index.tsx' file: Error: ENOENT: no such file or directory, open '<clutch>/frontend/packages/react-router-dom/index.tsx'
 @ ../core/dist/navigation.js 3:0-9:26 19:21-34 20:32-49 34:25-43 68:42-59 90:0-82 90:0-82
 @ ../core/dist/index.js 21:0-82 49:0-102:2 49:0-102:2 49:0-102:2 49:0-102:2
 @ ./src/index.jsx 5:0-42 9:52-61

WARNING in ../../node_modules/react-router/index.js
Module Warning (from ../../node_modules/react-scripts/node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '<clutch>/frontend/packages/react-router/index.tsx' file: Error: ENOENT: no such file or directory, open '<clutch>/frontend/packages/react-router/index.tsx'
 @ ../../node_modules/react-router-dom/index.js 1:0-102 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 5:0-312 102:23-29 134:23-29 154:6-13 155:10-21 156:10-21 157:10-25 197:10-21 198:10-25 241:2-12 257:10-21 270:10-21
 @ ../core/dist/navigation.js 3:0-9:26 19:21-34 20:32-49 34:25-43 68:42-59 90:0-82 90:0-82
 @ ../core/dist/index.js 21:0-82 49:0-102:2 49:0-102:2 49:0-102:2 49:0-102:2
 @ ./src/index.jsx 5:0-42 9:52-61
```

Reply:
For posterity, lets just confirm that without sourcemaps aren't required for tracing / debugging in dev

Reply:
I checked, couldn't find any differences with it on/off in dev, so I think we're good